pkg/config/config.go

```diff
@@ -153,7 +158,7 @@ func ParseClusterConfigFile(configPath string) (*types.ClusterConfig, error) {
 	}
 	ret, err := ParseClusterConfig(string(configBytes))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error parsing config file %s: %s", configPath, err.Error())
```

Comment:
Having the error message start with error stutters.  It goes "Error: error doing thing".  The fact that it is an error is implied by the log level and prefix.  Maybe replace with "could not parse %s: %s"?

Reply:
Done